spec/support/factories.cr

```diff
@@ -352,10 +352,13 @@ def tmp_path
   Shards::Specs.tmp_path
 end
 
-def run(command, *, env = nil)
+def run(command, *, env = nil, clear_env = false)
```

Comment:
`clear_env` is misleading since it's not what it does.

```suggestion
def run(command, *, env = nil, clear_crystal_opts = false)
```

Reply:
I'd like the name to reflect the intention, not the actual implementation. We might need to clear other variables as well at some point.